libs/blocks/modal/modal.css

```diff
@@ -203,7 +198,11 @@
   .dialog-modal.upgrade-flow-modal {
     border-radius: 0;
   }
-  
+
+  .dialog-modal.commerce-frame .fragment,
```

Comment:
This rule applies to all the nested fragments. Can you make them more explicit ?
  .dialog-modal.commerce-frame > .fragment,
.dialog-modal.commerce-frame > .section {